llvm/lib/Support/LockFileManager.cpp

```diff
@@ -94,7 +94,7 @@ static std::error_code getHostID(SmallVectorImpl<char> &HostID) {
   StringRef UUIDRef(UUIDStr);
   HostID.append(UUIDRef.begin(), UUIDRef.end());
 
-#elif LLVM_ON_UNIX
+#elif !defined(__wasi__)
```

Comment:
You're changing behavior for `!LLVM_ON_UNIX` (like Windows) to use this branch now. That seems probably unintended. Meant `#elif LLVM_ON_UNIX && !defined(__wasi__)`?

Reply:
Thanks, this is the cause of buildbot failures that I missed during my own review.